django/contrib/admin/static/admin/js/admin/RelatedObjectLookups.js

```diff
@@ -120,7 +120,7 @@
         var selects = django.jQuery(selectsSelector);
         selects.find('option').each(function() {
             if (this.value === objId) {
-                this.innerHTML = newRepr;
+                this.innerText = newRepr;
```

Comment:
> I think this may be a separate issue. Could you check if test_inline_char_fk_edit_with_popup fails on 1.8?

It does not pass.

```
======================================================================
FAIL: test_inline_char_fk_edit_with_popup (admin_views.tests.SeleniumAdminViewsChromeTests)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/Users/a1/projects/django/tests/admin_views/tests.py", line 4042, in test_inline_char_fk_edit_with_popup
    self.assertEqual(str(inner_text), str(parent.name))
AssertionError: 'w=' != 'w=<a;&\\"lsmvxg>\'rb-'
- w=
+ w=<a;&\"lsmvxg>'rb-
```

```
======================================================================
FAIL: test_inline_char_fk_edit_with_popup (admin_views.tests.SeleniumAdminViewsChromeTests)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/Users/a1/projects/django/tests/admin_views/tests.py", line 4043, in test_inline_char_fk_edit_with_popup
    self.assertEqual(str(inner_text), str(parent.name))
AssertionError: '"-o\\\'b;r=>w' != '"-o\\\'b;r=>w<kpq&uey'
- "-o\'b;r=>w
+ "-o\'b;r=>w<kpq&uey
?            ++++++++
```